components/profile/ProfileDashboard.tsx

```
@@ -372,33 +380,34 @@ const ApprenticeshipTrackerSection: React.FC<{ userId: string }> = ({ userId })
         status: 'Applied',
         notes,
       });
-      if (created) {
-        // Update apprenticeships state
-        setApprenticeships((prev) => [created, ...prev].sort((a, b) => new Date(b.updated_at || b.started_at).getTime() - new Date(a.updated_at || a.started_at).getTime()));
+      if (created && created.id) { // Ensure created object is valid
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)